src/common/fastopt.h

```diff
@@ -367,12 +367,17 @@ class FastOpt {
   }
 
   const FastOpt& operator[](const char* const key) const {
+#ifdef __APPLE__
```

Comment:
Why is this change needed? And if it is needed, can we always use the __APPLE__ path? It seems identical.

Reply:
Same as the above issue. I do believe we can use the linux codepath everywhere.

Reply:
If the same code path works everywhere, then let's do that.